packages/rest/src/__tests__/unit/ajv.service.unit.ts

```diff
@@ -39,6 +39,53 @@ describe('Ajv service', () => {
     expect(result).to.be.true();
   });
 
+  // possible values for type any
+  const TEST_VALUES = {
+    string: 'abc',
+    number: 123,
+    object: {random: 'random'},
+    array: [1, 2, 3],
+  };
+
+  context('accepts any type', () => {
+    for (const v in TEST_VALUES) {
+      testAnyTypeWith(v);
+    }
+
+    function testAnyTypeWith(value: string) {
+      it(`with value ${value}`, async () => {
+        const ajv = new AjvProvider().value();
+        const validator = ajv.compile({});
+        const result = await validator(value);
+        expect(result).to.be.true();
+      });
+    }
+  });
+
+  context('accepts any type - property', () => {
+    for (const v in TEST_VALUES) {
+      testAnyTypeWith(v);
+    }
+
+    function testAnyTypeWith(value: string) {
+      it(`with value ${value}`, async () => {
+        const ajv = new AjvProvider().value();
+        const validator = ajv.compile({
+          type: 'object',
+          properties: {
+            name: {type: 'string'},
+            arbitraryProp: {},
```

Comment:
Can we also test `arbitraryProp: true`?

Reply:
cc @raymondfeng see the comment in https://github.com/strongloop/loopback-next/pull/4930#discussion_r398970633